src/hotspot/share/runtime/globals.hpp

```diff
@@ -1282,6 +1282,11 @@ const int ObjectAlignmentInBytes = 8;
           "(0 means none)")                                                 \
           range(0, max_jint)                                                \
                                                                             \
+  product(intx, ServiceThreadCleanupInterval, 1000, DIAGNOSTIC,             \
+          "Wake the ServiceThread to do periodic cleanup checks"            \
+          "(0 means none)")                                                 \
+          range(0, max_jint)                                                \
```

Comment:
The time unit needs to be mentioned - I assume it is ms?

Reply:
Yes it is ms, like guaranteed safepoint interval, actually PlatformMonitor.lock units, which is ms.  I wonder if this is a useful diagnostic option though.  Maybe we shouldn't have it?